api/document/js/__tests__/schema.test.ts

```diff
@@ -23,7 +23,7 @@ describe('unimplemented_node', () => {
 });
 
 describe('heading', () => {
-  [2, 3, 4, 5].forEach(depth => {
+  [2, 3, 4, 5].forEach((depth) => {
```

Comment:
This lint error actually comes from a recently-merged PR, but it wasn't checked in that PR because of a problem w/ the `lint` script in the `package.json`, which caused the linter to skip this file. I've fixed that problem in this PR though, which made the linter catch this.